polkadot-parachains/Cargo.toml

```
@@ -1,12 +1,13 @@
 [package]
-name = "polkadot-collator"
-version = "5.4.0"
+name = "polkadot-parachain"
+version = "0.9.19"
```

No, why would that be?
In the future, will we have a new version of the `polkadot-parachain` for each new version of polkadot ? and never in between ?
The answer maybe "kinda yes today" but that is not the plan and this is why the collator version is decoupled from polkadot.

```suggestion
version = "5.5.0"
```

---

Less of an issue once polkadot reaches `1.0`. For now we can do `0.9.19-patch1` if we really need to. I get the 'in theory we might need to do a patch', but we have a lot of users and this makes things simpler if we in general we keep the version numbers in check. If my polkadot parachain collator version is matching the relay chain version I get a warm fuzzy feeling inside that I just don't get with `v5.5`.

---

Because (a) the Cumulus node contains a Polkadot node, so having a major Cumulus release doesn't make sense, and (b) it was a bad change [in the first place](https://github.com/paritytech/cumulus/pull/649/files#diff-645962a0e1b239c8d20e814dcdb19ea39e7d9ab3adb51d59b4abfd7571f1be65R3), bumping from 0.1.0 to 4.0.0.

We shouldn't have a 1.0.0 client release at all.

---

> We cannot go back in terms of release number.

Why not? Very few people are running nodes now. If there's a time to do it, it's now.

---

It's got a different binary name so it's allowed a different version number. Windows is now on 11. I remember when it was 95...

---

Remember it's only statemine/t that use this directly. I've checked a lot of other parachain repos and they have no mention of `polkadot-collator`.

---

Yeah but others will (like Alliance, governance, etc)